vocabularies/UI.xml

```diff
@@ -1687,6 +1687,51 @@ This is in contrast to [DataFieldForIntentBasedNavigation](#DataFieldForIntentBa
         <Annotation Term="Core.Description" String="The contents of this property must not be propagated to the app-to-app navigation context" />
       </Term>
 
-    </Schema>
+      <Term Name="UserInteraction" Type="UI.UserInteractionType" Nullable="false" AppliesTo="NavigationProperty">
+        <Annotation Term="Core.Description">
+          <String>When the annotated navigation property or its `odata.navigationLink` occurs in a response,
+          the UI shall use the addressed entity or collection to immediately interact with the user
+          and then repeat the corresponding request with the additional information obtained from the user</String>
+        </Annotation>
+        <Annotation Term="Core.LongDescription">
+          <String>If the response is an [error response](https://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.html#sec_ErrorResponse),
+          the annotated navigation property MAY occur in the `innererror` property which has the format of a [message body](https://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.html#sec_MessageBody).
```

Comment:
Instead of `innererror`, we could use a new instance annotation of type `Edm.Untyped`, but this would have to come from a new vocabulary that uses version `Edmx Version="4.01"`.

Reply:
I'd prefer a dedicated instance annotation, the `innererror` is intended for debugging information.